feed.xml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
We have jekyll-feed available, can it do this?  https://github.com/jekyll/jekyll-feed